keyform/urls.py

```diff
@@ -4,6 +4,9 @@
 
 urlpatterns = [
     url(r'^$', views.HomeView.as_view(), name='home'),
+    url(r'^contact', views.ContactView.as_view(), name='contact'),
+    url(r'^edit_contact/(?P<pk>\d+)$', views.EditContactView.as_view(), name='edit_contact'),
```

Comment:
We should stick with a simmilar style for multi word url's 
`edit-contact` vs `edit_contact`